src/mplfinance/plotting.py

```diff
@@ -267,7 +307,9 @@ def plot( data, **kwargs ):
             mavc = None
             
         for mav in mavgs:
-            mavprices = data['Close'].rolling(mav).mean().values 
+            mavprices = data['Close'].rolling(mav).mean().values
+            if ptype == 'renko':
```

Comment:
I don't think this is the correct way to handle `mav` for renko.  

[This document](https://school.stockcharts.com/doku.php?id=chart_analysis:renko), in the section entitled **"Trends, Support and Resistance"** seems to indicate that renko chartists like to look at the moving average *of the renko bricks prices themselves*.  (Whereas the way this is coded the mav is based off the user-input Close price).

Also, according to the document, the **mav period is the number of renko values.**  I'm not sure if that means having to count repeated dates as one??  From the document:  "Note that a 10-period moving average calculation is based on the last ten Renko values, not the last ten trading days."

Reply:
You're definitely right about the mav calculation, I must have missed that part when researching Renko charts.  I will work to adjust this now!